src/assessment.js

```diff
@@ -27,8 +26,8 @@ var q1 = /* your answer here */;
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-var q2 = [/* your answer here */];
-//
+var q2 = [fire];
+//laserQuantity is defined WITHIN the scope of deathStar, so it can be modified or used within that scope, but deathStar can't do anything with it before the point that it is defined.
```

Comment:
Though you're technically correct that `laserQuantity` can't be used until it's defined, that doesn't mean it's not in the scope. Consider the variables that might be inside `fire` - are they not part of the `fire` scope? If not, what scope _are_ they in?

Obviously, though I disagree with your reasoning, it's clear that you understand the concept, and I will not deduct any points.